modules/cudf/src/data_frame.ts

```diff
@@ -884,4 +884,35 @@ export class DataFrame<T extends TypeMap = any> {
     });
     return new DataFrame(series_map);
   }
+
+  /**
+   * Creates a DataFrame of `BOOL8` Series where `true` indicates the value is `null` or `NaN` and
+   * `false` indicates the value is valid.
+   *
+   * @returns DataFrame<Bool8> with series of `BOOL8` elements with `true` representing `null` or
+   * `NaN` values
+   *
+   * @example
+   * ```typescript
+   * import {DataFrame, Series, Int32, Float32}  from '@rapidsai/cudf';
+   * const df = new DataFrame({
+   *  a: Series.new({type: new Int32, data: [0, 1, 2, 3, 4, 4]}),
+   *  b: Series.new({type: new Float32, data: [0, NaN, 2, 3, 4, 4]})
+   * });
+   *
+   * df.isNull()
+   * ```
+   */
+  isNull(): DataFrame<T> {
+    const temp       = new Table({columns: this.select(this.names)._accessor.columns});
+    const series_map = {} as SeriesMap<T>;
+    this._accessor.names.forEach((name, index) => {
+      if (this.get(name) instanceof Float32Series || this.get(name) instanceof Float64Series) {
+        series_map[name] = Series.new(temp.getColumnByIndex(index).isNaN()) as any;
```

Comment:
I am forced to cast this as `any` as I get this IDE error otherwise

```
Type 'Bool8Series' is not assignable to type 'AbstractSeries<T[string & keyof T]>'.
  Types of property '_col' are incompatible.
    Type 'Column<Bool8>' is not assignable to type 'Column<T[string & keyof T]>'.
      Type 'Bool8' is not assignable to type 'T[string & keyof T]'.ts(2322)
```

I feel like this could be fixed by extending the `Bool8Series` type into another one of the parameters, but, my knowledge of how types inherit in this app is a bit lacking, any suggestions?